lib/membrane_mp4/payloader/h264.ex

```diff
@@ -128,4 +134,7 @@ defmodule Membrane.MP4.Payloader.H264 do
   defp encode_parameter_sets(pss) do
     Enum.map_join(pss, &<<byte_size(&1)::16-integer, &1::binary>>)
   end
+
+  defp fetch_parameters_set([] = _new_ps, ps), do: ps
+  defp fetch_parameters_set(ps, _old_ps), do: ps
```

Comment:
```suggestion
  defp fetch_parameters_sets([] = _new_ps, ps), do: ps
  defp fetch_parameters_sets(ps, _old_ps), do: ps
```

Reply:
It is actually a single parameter set 😛 (either sps or pps).

Reply:
I think it doesn't have to be, we retrieve a list [here](https://github.com/membraneframework/membrane_mp4_plugin/pull/76/files#diff-6106dedc0d1aad514f4602eedffb94b1299781db253a77ab1ebe4eb2b2b6ba46R54), also there can be multiple sps and multiple pps in AVCC, also there's [encode_parameter_sets](https://github.com/membraneframework/membrane_mp4_plugin/pull/76/files#diff-6106dedc0d1aad514f4602eedffb94b1299781db253a77ab1ebe4eb2b2b6ba46R134)